cmd/svcat/class/get_cmd.go

```diff
@@ -69,6 +79,10 @@ func (c *getCmd) Validate(args []string) error {
 			c.name = args[0]
 		}
 	}
+	// Just for now while I get PR reviewed
```

Comment:
Seems like instead of returning nil, this should return an error about not setting both flags at once.